### DISCUSSION TITLE
Add support of repeating interval to scheduled jobs

### DISCUSSION THREAD

**Discussion by rkuzmych** [jobrunr/jobrunr#364] [Ideas]
Hello!

It would be nice to have out of box support of repeating interval. Example:

We scheduled job to start at 15:40:00 2/22/2022 and we want to repeat this job every 15min. 
1st triggered: 15:40:00 -> 2nd trigerred: 15:55:00 -> 3rd triggered: 16:10:00 -> ... and so on util job will be deleted.

**Comment by rdehuyss** [comment]
This will be supported in v5.
